test/suite/test_chunkcache03.py

```diff
@@ -119,11 +119,27 @@ def test_chunkcache03(self):
         self.assertGreater(total_chunks, 0)
         self.assertGreater(pinned_chunks, 0)
 
-        # Assert that pinned chunks make up 50% of the total chunks.
-        self.assertEqual(total_chunks/pinned_chunks, 2)
+        # Assert that pinning a table also pins its backing chunks.
+        # We can't measure this directly, so exploit the fact that all four tables have been
+        # populated to roughly the same size and will use ~25% of available chunks each.
 
-        # Reconfigure wiredtiger and mark the pinned objects as unpinned and vice-versa.
-        self.conn.reconfigure('chunk_cache=[pinned=("table:chunkcache03", "table:chunkcache04")]')
+        # We start with two tables pinned. This is half the total chunks
+        self.assertEqual(round(total_chunks/pinned_chunks), 2)
```

Comment:
As there are no digits of precision, you could use the floor division operator:
```suggestion
        self.assertEqual(total_chunks // pinned_chunks, 2)
```
Just a fyi: personally I think both are fine.

Reply:
I considered it initially, but while we haven't seen it happen I wanted to protect against values like 1.9875 as well.

Reply:
Thank you for clarifying.